monitoring/README.md

```
@@ -0,0 +1,71 @@
+```yaml
+rule_files:
+  - "/etc/prometheus/alerts/*.yml"
+```
+
+Ensure the contents of `monitoring/alerts/` are mapped to that directory.
+
+### 2. Grafana Provisioning
+
+To automatically load the data sources and dashboards:
+
+1.  Copy `provisioning/datasources.yml` to your Grafana `/etc/grafana/provisioning/datasources/` directory.
```

markdownlint-cli2 (0.22.1)

[warning] Spaces after list markers
Expected: 1; Actual: 2

(MD030, list-marker-space)